README.md

```diff
@@ -13,5 +13,17 @@ mkdir /home/cheriot-rtos.mapped
 sudo bindfs --map=martha/boris /home/cheriot-rtos /home/cheriot-rtos.mapped
 ```
 
-VScode can now be launched with a dev container from /home/cheriot-rtos.mapped 
+VScode can now be launched with a dev container from /home/cheriot-rtos.mapped
+
+## Building locally
+
+One way to test the dev container is to use podman.
+Clone this repo and then run the following commands from within it:
+
+```sh
+podman build --tag 'cheriot-devcontainer-sonata-sim' .
+podman run -ti localhost/cheriot-devcontainer-sonata-sim:latest
```

Comment:
`-ti` is normally written `-it`.  You probably also want `--rm` or it will leave the container running.

Reply:
Added both of these.